programs/leap-util/actions/generic.cpp

```diff
@@ -4,7 +4,14 @@
 
 void generic_actions::setup(CLI::App& app) {
    auto* sub = app.add_subcommand("version", "retrieve version information");
-   sub->require_subcommand(1);
+
+   /// makes  subcommand "version" non-actionable, actions expected in child subcommands
+   sub->require_subcommand(); 
+   
+   /// this will make options defined on parent subcommand to be actionable for child subcommands and to be displayed in --help call for them
+   /// it's not required in this handler, but set here for documentation purposes to showcase that in most cases for custom action handlers it is a desirable behavior
+   sub->fallthrough();     
```

Comment:
This should be added for the other top-level commands too, right?

Reply:
Great suggestion, added!